.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "editor.codeActionsOnSave": {
+    "source.fixAll.eslint": true
+  },
+  "editor.formatOnSave": false
```

Comment:
Adds vscode settings to disable prettier's format on save and use eslint rules instead.